### ISSUE TITLE
Sometimes meet a ECONNRESET error during hexo generate, is there anything needs network?

### ISSUE THREAD

**Issue by zombie110year** [hexojs/hexo#4149]
## Check List

Please check followings before submitting a new issue.

- [x] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
- [x] I have already searched existing issues and they are not help  to me
- [x] I examined error or warning messages and it's difficult to solve
- [x] Using [the latest](https://github.com/hexojs/hexo/releases) version of Hexo (run `hexo version` to check)
- [x] Node.js is higher than 8.6.0

## Expected behavior

`hexo g` and all pass.

## Actual behavior

meet `ECONNRESET` randomly:
```
events.js:298
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:205:27)
Emitted 'error' event on ClientRequest instance at:
    at TLSSocket.socketErrorListener (_http_client.js:432:9)
    at TLSSocket.emit (events.js:321:20)
    at emitErrorNT (internal/streams/destroy.js:84:8)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}
```

## How to reproduce?

I don't know, It randomly appears and sometimes works well.
use command `hexo g`.

## Environment & Settings

**Node.js & npm version**

```
hexo: 4.2.0
hexo-cli: 2.0.0
os: Windows_NT 10.0.19041 win32 x64
node: 13.8.0
v8: 7.9.317.25-node.28
uv: 1.34.1
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 79
nghttp2: 1.40.0
napi: 5
llhttp: 2.0.4
openssl: 1.1.1d
cldr: 36.0
icu: 65.1
tz: 2019c
unicode: 12.1
```

# Others

I'm living in China mainland, maybe there were something blocked by GFW which is necessary by hexo.

If anyone could teach me some skills to research network problem, I might be able to provide more infomations.

**Comment by stefanJi** [comment]
I met this.

```
➜  hexo server
INFO  Start processing
INFO  Hexo is running at http://localhost:4000/. Press Ctrl+C to stop.
events.js:173
      throw er; // Unhandled 'error' event
      ^

Error: connect ECONNREFUSED 13.250.168.23:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1083:14)
Emitted 'error' event at:
    at TLSSocket.socketErrorListener (_http_client.js:397:9)
    at TLSSocket.emit (events.js:197:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at processTicksAndRejections (internal/process/next_tick.js:76:17)
```
Network request capture:
<img width="1126" alt="Screen Shot 2020-03-01 at 00 36 05" src="https://user-images.githubusercontent.com/15273198/75611335-a6150f80-5b54-11ea-88ac-6965c4d04cc7.png">

**Comment by RivenNero** [comment]
I met this too.
```
events.js:288
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:205:27)
Emitted 'error' event on ClientRequest instance at:
    at TLSSocket.socketErrorListener (_http_client.js:426:9)
    at TLSSocket.emit (events.js:311:20)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ECONNRESET',
  code: 'ECONNRESET',
  syscall: 'read'
}
```

**Comment by stevenjoezhang** [comment]
Are you using theme NexT with `reminder` option enabled in `_config.yml`?

**Comment by dsanying** [comment]
> 
> 
> Are you using theme NexT with `reminder` option enabled in `_config.yml`?

I have the same problem of 
```
events.js:288
      throw er; // Unhandled 'error' event
      ^
```
and i am using the reminder option,so how to solve this question?

**Comment by stevenjoezhang** [comment]
Fixed in https://github.com/theme-next/hexo-theme-next/commit/9b543ddacf21a59f2daa52b0ae64075aed62fca2

**Comment by dsanying** [comment]
do I need to clone the theme again or don't need to do anything?

**Comment by zombie110year** [comment]
thanks!
